how-we-work/oncall-responsibilities.md

```diff
@@ -20,13 +20,13 @@ An on-call rotation begins on Monday at 9am PT. Each week, there is a primary an
+| repo                    | dev                            | staging                                                                                  | prod                                                                                     | dev-canary                                                |
+| ----------------------- | ------------------------------ | ---------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- | --------------------------------------------------------- |
+| single-cell-infra       | TFE auto                       | TFE auto (excl. [explorer](https://github.com/chanzuckerberg/single-cell-infra/blob/a8d1a3cc5f36280de69f7250f4a6422a55d574fc/terraform/tfe/locals.tf.json#L21) infra)                                                                                 | TFE manual                                                                               | TFE manual                                                |
+| single-cell-data-portal | Need to set up auto deployment | Github Action Auto Deploy                                                                | happy/script Deploy Manual                                                               | no deployment                                             |
+| single-cell-explorer    | Github Action Auto Deploy      | single-cell-infra deploy script to kick off github action (also possible to run locally) | single-cell-infra deploy script to kick off github action (also possible to run locally) | Github Action Auto Deploy (does not auto rebase off main) |
 
-The principal responsibility of the primary on-call is to coordinate deployments on Wednesday (from `staging` to `prod`) and Thursday (from `main` to `staging`). Both the cellxgene Data Portal and the cellxgene Explorer need to be deployed and require two different processes. On both days, please try to promote by 1pm PST.
+The principal responsibility of the primary on-call is to coordinate deployments on Wednesday (from `staging` to `prod`, 6 days after previous week's staging deploy of the same release), and Thursday (from `main` to `staging`). Both the cellxgene Data Portal and the cellxgene Explorer need to be deployed and require two different processes. On both days, please try to promote by 10am PT/1pm ET.
 
 The instructions to deploy `hosted-cellxgene` (a.k.a. cellxgene Explorer) can be found [here](https://github.com/chanzuckerberg/single-cell-infra/tree/main/terraform/modules/hosted-cellxgene#redeploying-the-application). The instructions to deploy `corpora-data-portal` (a.k.a. cellxgene Data Portal) can be found [here](https://github.com/chanzuckerberg/single-cell-infra/tree/main/terraform/modules/corpora#redeploying-the-application).
```

Comment:
Are these links still accurate for deployment instructions?

Reply:
still unclear on that! when I pair w/timmy today on deploys, will check and update as needed.  we can keep PR open until then...